pkg/codegen/python/gen.go

```diff
@@ -347,7 +347,7 @@ func (mod *modContext) genAwaitableType(w io.Writer, obj *schema.ObjectType) str
 	baseName := pyClassName(tokenToName(obj.Token))
 
 	// Produce a class definition with optional """ comment.
-	fmt.Fprintf(w, "class %s:\n", baseName)
+	fmt.Fprintf(w, "\nclass %s:\n", baseName)
```

Comment:
Nit: Instead of printing the `\n` in front of `class`, what do you think of having a single `fmt.Fprintf(w, "\n")` call before the call to `mod.genAwaitableType` on line 686 below?

Reply:
Sure. I've made similar changes previously so I'm going to fix them to also follow this pattern.